syft/workers/base.py

```diff
@@ -116,11 +116,11 @@ def __init__(
         self._message_router = {
             CommandMessage: self.execute_command,
             PlanCommandMessage: self.execute_plan_command,
-            ObjectMessage: self.set_obj,
+            ObjectMessage: self.handle_object_msg,
             ObjectRequestMessage: self.respond_to_obj_req,
             ForceObjectDeleteMessage: self.rm_obj,  # FIXME: there is no ObjectDeleteMessage
```

Comment:
This method on object_storage doesn't expect a message

Reply:
`set_obj` doesn’t, so I added a new method that does.